code/process_sequences.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -euo pipefail
```

Comment:
Per specification 2, you should refactor this bash script to take the additional parameter 'ALL' as a second command line parameter. When you run the script with ALL, the script will output everything as you had it originally.

Reply:
@mhaliz Thanks for the feedback. I've done some digging, but I'm still not sure how to implement this. Would you suggest adding the `@$` parameter? Not sure which line either.

Reply:
@SpaceGold use a nested `if` or a compound logical test, eg something like: this == that & thing2 == pqr